native/cocos/platform/apple/modules/CanvasRenderingContext2DDelegate.mm

```diff
@@ -72,8 +72,8 @@ @interface CanvasRenderingContext2DDelegateImpl : NSObject {
     cc::Data        _imageData;
     NSBezierPath *  _path;
 
-    CanvasTextAlign      _textAlign;
-    CanvasTextBaseline   _textBaseLine;
+    cc::ICanvasRenderingContext2D::CanvasTextAlign      _textAlign;
```

Comment:
Can use CanvasTextAlign?

Reply:
I don't know much about object-c. I tried using, but it didn't seem to work

Reply:
Can't.
Because it is in objective-c class.